src/main/java/com/example/movieflowapi/controller/MovieController.java

```diff
@@ -0,0 +1,20 @@
+package com.example.movieflowapi.controller;
+
+import com.example.movieflowapi.model.dto.Movie;
+import com.example.movieflowapi.repository.MovieDAO;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class MovieController {
+    private final MovieDAO dao;
+
+    @GetMapping(value = "/comedies")
+    public List<Movie> getComedies(@RequestParam(value = "names") List<String> names){
```

Comment:
Better to mention the fact that this method is returning values based on actor names, so it will be something like getComediesByActorName + lets rename param 'names' to 'actorNames' to make it more clear